src/vxworks/mod.rs

```diff
@@ -0,0 +1,2322 @@
+//! Hacking together the definitions for VxWorks Bindings
+use core::ptr::null_mut;
+use core::mem::size_of;
+
+#[cfg_attr(feature = "extra_traits", derive(Debug))]
+pub enum DIR{}
+impl ::Copy for DIR {}
+impl ::Clone for DIR {
+    fn clone(&self) -> DIR { *self }
+}
+
+/*
```

Comment:
I think its ok to just remove these then. No need to mention them.

Reply:
Sure.